spec/factories.rb

```diff
@@ -0,0 +1,26 @@
+FactoryGirl.define do
+  factory :challenge do
+    title :test
+    description :test
+    input :a
+    output :b
+    diff :c
+    user
+  end
+
+  factory :entry do
+    script :a
+    created_at Time.now
```

Comment:
small nit; when we create `Entry`s they're in utc. shouldn't matter for the factory here, but might be nice to keep it consistent